setup.py

```diff
@@ -41,7 +41,8 @@ def run_tests(self):
 packages = ['requests']
 
 requires = [
-    'chardet>=3.0.2,<5',
+    'charset_normalizer~=1.4.1; python_version >= "3"',
```

Comment:
Do we need to update this with v2.0.0?

Reply:
We haven't tested v2. I'd rather get something imperfect out than something perfect. Today's literally the only time I've had to review anything

Reply:
Makes sense, I/we can handle the upgrade in a follow-up PR.